src/WebJobs.Script/Config/FunctionsHostingConfigOptions.cs

```diff
@@ -31,6 +31,28 @@ public bool FunctionsWorkerDynamicConcurrencyEnabled
             }
         }
 
+        /// <summary>
+        /// Gets the highest version of extension bundle v4 supported
+        /// </summary>
+        public string MaximumSupportedBundleV3Version
+        {
+            get
+            {
+                return GetFeature(ScriptConstants.MaximumSupportedBundleV3Version) ?? "3.19.0";
+            }
+        }
+
+        /// <summary>
+        /// Gets the highest version of extension bundle v3 supported
```

Comment:
I think the comments are swapped on these two properties - this should say v4?

Reply:
Fixed